RecoEgamma/EgammaElectronProducers/src/LowPtGsfElectronFeatures.cc

```diff
@@ -164,6 +164,400 @@ namespace lowptgsfeleid {
+                                 eid_shape_full5x5_HoverE,
+                                 eid_trk_nhits,
+                                 eid_trk_chi2red,
+                                 eid_gsf_chi2red,
+                                 eid_brem_frac,
+                                 eid_gsf_nhits,
+                                 eid_match_SC_EoverP,
+                                 eid_match_eclu_EoverP,
+                                 eid_match_SC_dEta,
+                                 eid_match_SC_dPhi,
+                                 eid_match_seed_dEta,
+                                 eid_sc_E,
+                                 eid_trk_p,
+                                 gsf_mode_p,
+                                 core_shFracHits,
+                                 gsf_bdtout1,
+                                 gsf_dr,
+                                 trk_dr,
+                                 sc_Nclus,
+                                 sc_clus1_nxtal,
+                                 sc_clus1_dphi,
+                                 sc_clus2_dphi,
+                                 sc_clus1_deta,
+                                 sc_clus2_deta,
+                                 sc_clus1_E,
+                                 sc_clus2_E,
+                                 sc_clus1_E_ov_p,
+                                 sc_clus2_E_ov_p};
+    return output;
+  }
+
+  ////////////////////////////////////////////////////////////////////////////////
+  //
+  std::vector<float> features(edm::Ref<std::vector<reco::GsfElectron> > const& ele, float rho, float unbiased) {
+    return features(edm::refToPtr(ele), rho, unbiased);
+  }
+
+  ////////////////////////////////////////////////////////////////////////////////
+  //
+  std::vector<float> features(edm::Ref<edm::View<reco::GsfElectron> > const& ele, float rho, float unbiased) {
+    return features(edm::refToPtr(ele), rho, unbiased);
+  }
+
+  ////////////////////////////////////////////////////////////////////////////////
+  // feature list for original models (2019Aug07 and earlier)
+  std::vector<float> features_V0(edm::Ptr<reco::GsfElectron> const& ele, float rho, float unbiased) {
+    float eid_rho = -999.;
+    float eid_sc_eta = -999.;
+    float eid_shape_full5x5_r9 = -999.;
+    float eid_sc_etaWidth = -999.;
+    float eid_sc_phiWidth = -999.;
+    float eid_shape_full5x5_HoverE = -999.;
+    float eid_trk_nhits = -999.;
+    float eid_trk_chi2red = -999.;
+    float eid_gsf_chi2red = -999.;
+    float eid_brem_frac = -999.;
+    float eid_gsf_nhits = -999.;
+    float eid_match_SC_EoverP = -999.;
+    float eid_match_eclu_EoverP = -999.;
+    float eid_match_SC_dEta = -999.;
+    float eid_match_SC_dPhi = -999.;
+    float eid_match_seed_dEta = -999.;
+    float eid_sc_E = -999.;
+    float eid_trk_p = -999.;
+    float gsf_mode_p = -999.;
+    float core_shFracHits = -999.;
+    float gsf_bdtout1 = -999.;
+    float gsf_dr = -999.;
+    float trk_dr = -999.;
```

Comment:
```suggestion
    float gsf_dr = 5;
    float trk_dr = 5;
```
given what's done with out of range, these seem like more appropriate defaults

... same for the non-V0 case above

Reply:
I think I'd prefer to leave this as is.

Reply:
I'd also prefer to leave it like this @cavalari